cmd/autoheal/object_template_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+)
+
+func TestNewObjectTemplateBuilder(t *testing.T) {
+	b := NewObjectTemplateBuilder()
+	if b.right != "}}" {
```

Comment:
I think it is better to avoid accessing private fields in the tests, as that means that we can't change the internals without also changing the tests. Instead of doing this you can try to process something that requires the delimiters, and then check that they did work. Same for variables or any other internal implementation detail: tests should not use them, in my opinion.